python/cugraph/cugraph/dask/common/mg_utils.py

```diff
@@ -107,6 +107,9 @@ def start_dask_client():
 
     visible_devices = ",".join([str(i) for i in range(1, n_devices + 1)])
 
+    print(
+        "************************** starting DASK ************************************"
+    )
```

Comment:
We should probably remove this debug print.  This would be good if/when we add proper logging and the user sets the loglevel to debug though.

Reply:
I forgot that was added.   it was there for testing a side dask issue starting the cluster - its removed